liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/BasicSupport.java

```diff
@@ -346,7 +344,7 @@ protected void installFromFile() throws Exception {
     protected void installFromArchive() throws Exception {
         InstallLibertyTask installTask = (InstallLibertyTask) ant.createTask("antlib:net/wasdev/wlp/ant:install-liberty");
         if (installTask == null) {
-            throw new NullPointerException("install-liberty task not found");
+            throw new IllegalStateException("install-liberty task not found");
```

Comment:
While we are here, we could make this a bit more informative.  How about:

The Liberty install-liberty Ant task was not found. Verify that the required version of the wlp-anttasks.jar is available.